docs/kb/developer-resources/finding-your-api-token-and-app-id.md

```diff
@@ -0,0 +1,32 @@
++++
+date = "2017-03-13T00:00:00Z"
+lastmod = "2017-03-13T00:00:00Z"
+title = "Finding Your API Token and App ID"
+weight = "999999"
+categories = [ "Knowledgebase", "Developer Resources" ]
++++
+
+In order to utilize many of the developer API endpoints you will need to know your API Token as well as other items such as your App ID or Channel ID. This document will provide examples of how those values can be located using the curl utility.
+
+## API Token
+
+You can create and show your API Token inside your Vendor Web portal by clicking on the Teams & Tokens link.
```

Comment:
This should include the url since its linkable https://vendor.replicated.com/#/team